pkg/analyzer/pod.go

```diff
@@ -60,7 +60,7 @@ func (PodAnalyzer) Analyze(a common.Analyzer) ([]common.Result, error) {
 		// Check through container status to check for crashes or unready
 		for _, containerStatus := range pod.Status.ContainerStatuses {
 			if containerStatus.State.Waiting != nil {
-				if containerStatus.State.Waiting.Reason == "CrashLoopBackOff" || containerStatus.State.Waiting.Reason == "ImagePullBackOff" || containerStatus.State.Waiting.Reason == "CreateContainerConfigError" {
+				if containerStatus.State.Waiting.Reason == "CrashLoopBackOff" || containerStatus.State.Waiting.Reason == "ImagePullBackOff" || containerStatus.State.Waiting.Reason == "CreateContainerConfigError" || containerStatus.State.Waiting.Reason == "PreCreateHookError" || containerStatus.State.Waiting.Reason == "CreateContainerError" || containerStatus.State.Waiting.Reason == "PreStartHookError" || containerStatus.State.Waiting.Reason == "RunContainerError" || containerStatus.State.Waiting.Reason == "ImageInspectError" || containerStatus.State.Waiting.Reason == "ErrImagePull" || containerStatus.State.Waiting.Reason == "ErrImageNeverPull" || containerStatus.State.Waiting.Reason == "InvalidImageName" {
```

Comment:
I suggest to make it a variable for readability e.g ```
containerUnhealthyStatus := containerStatus.State.Waiting.Reason == "CrashLoopBackOff" || containerStatus.State.Waiting.Reason == "ImagePullBackOff" || containerStatus.State.Waiting.Reason == "CreateContainerConfigError" || containerStatus.State.Waiting.Reason == "PreCreateHookError" || containerStatus.State.Waiting.Reason == "CreateContainerError" || containerStatus.State.Waiting.Reason == "PreStartHookError" || containerStatus.State.Waiting.Reason == "RunContainerError" || containerStatus.State.Waiting.Reason == "ImageInspectError" || containerStatus.State.Waiting.Reason == "ErrImagePull" || containerStatus.State.Waiting.Reason == "ErrImageNeverPull" || containerStatus.State.Waiting.Reason == "InvalidImageName"```

Reply:
Thank you. @arbreezy 😄 

For more refactoring, I modified conditions.

Reply:
Nice good job @jmyung